server/src/pages/shotindex/view.js

```diff
@@ -148,6 +148,9 @@ class Body extends React.Component {
   renderErrorMessages() {
     return (
       <div>
+        <Localized id="">
```

Comment:
Why do the other Localized's have ids and this one doesn't?

Reply:
D'oh!  I deleted the id of some pasted code but didn't insert the new one.